homeassistant/components/mysensors/gateway.py

```diff
@@ -280,6 +283,12 @@ async def _gw_start(
     hass: HomeAssistantType, entry: ConfigEntry, gateway: BaseAsyncGateway
 ):
     """Start the gateway."""
+    gateway_ready = asyncio.Future()
```

Comment:
I think we should make this an `asyncio.Event` instead since we're not interested in the result from the future. Events are part of the higher level asyncio api while Futures are lower level api.